cli/package.json

```diff
@@ -1,16 +1,22 @@
 {
 	"name": "ic-mops",
 	"version": "0.6.1",
-	"type": "module",
+	"type": "commonjs",
```

Comment:
Can we keep this as `module`, because we build it with `esbuild` anyway?
If it is necessary just change inside `dist` dir?

Reply:
This is necessary due to current limitations with `pkg` (which reads the `package.json` file to determine how to compile the binaries), but this also provides the added benefit of supporting Node versions prior to 16.x.